command/agent/consul/client.go

```diff
@@ -675,11 +675,11 @@ func (c *ServiceClient) RegisterAgent(role string, services []*structs.Service)
 // serviceRegs creates service registrations, check registrations, and script
 // checks from a service. It returns a service registration object with the
 // service and check IDs populated.
-func (c *ServiceClient) serviceRegs(ops *operations, service *structs.Service, task *TaskServices) (
+func (c *ServiceClient) serviceRegs(ops *operations, service *structs.Service, task *WorkloadServices) (
```

Comment:
Maybe rename this `task` parameter? (Same for the other cases of this throughout this file.)